events-api/src/modules/events/dto/requests/list-events-params.dto.ts

```diff
@@ -1,5 +1,7 @@
+import { SortOrder } from './../../../../shared/constants/sort-order.constant';
 import { Transform } from 'class-transformer';
-import { IsInt, IsOptional, IsPositive, IsString, Max } from 'class-validator';
+import { IsEnum, IsInt, IsOptional, IsPositive, IsString, Max } from 'class-validator';
+import { SortBy } from 'src/shared/constants/sort-by.constant';
```

Comment:
Why it is located on the shared level, isn't is solely for Events module?

Reply:
Moved into events